Sources/Apollo/HTTPNetworkTransport.swift

```diff
@@ -343,13 +343,15 @@ public class HTTPNetworkTransport {
     let sendQueryDocument: Bool
     let autoPersistQueries: Bool
     switch operation.operationType {
-    case .query:
+    case .query, .mutation:
       if isPersistedQueryRetry {
-        useGetMethod = self.useGETForPersistedQueryRetry
+        useGetMethod = operation.operationType == .mutation ? false : self.useGETForPersistedQueryRetry
         sendQueryDocument = true
         autoPersistQueries = true
       } else {
-        useGetMethod = self.useGETForQueries || (self.enableAutoPersistedQueries && self.useGETForPersistedQueryRetry)
+        useGetMethod = operation.operationType == .mutation ? false :
+                                                   self.useGETForQueries ||
+                                                   (self.enableAutoPersistedQueries && self.useGETForPersistedQueryRetry)
```

Comment:
I'm not a giant fan of these huge ternaries, but I'll deal with that post-merge.